internal/pkg/manifest/validate_env.go

```diff
@@ -81,6 +84,56 @@ func (cfg environmentVPCConfig) validate() error {
 	return nil
 }
 
+// validate returns nil if securityGroupRule has all the required parameters set.
+func (cfg SecurityGroupRule) validate() error {
+	if cfg.CidrIP == "" {
+		return fmt.Errorf(`cidr`)
+	}
+	if cfg.IpProtocol == "" {
+		return fmt.Errorf(`ip_protocol`)
+	}
+	return cfg.PortsConfig.validate()
+}
+
+// validate if ports are set.
+func (cfg PortsConfig) validate() error {
+	if cfg.IsEmpty() {
+		return fmt.Errorf(`ports`)
+	}
+	if !cfg.Ports.IsEmpty() {
+		return cfg.Ports.validate()
+	}
+	return nil
+}
+
+// validate checks if PortsRangeBand is set correctly.
+func (str PortsRangeBand) validate() error {
+	ports := strings.Split(string(str), "-")
+	if len(ports) != 2 {
+		return fmt.Errorf("invalid ports value %s. Should be in format of ${from_port}-${to_port}", string(str))
```

Comment:
Typically we don't use capital letters in error messages or have punctuation: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Would you mind updating here and:
https://github.com/aws/copilot-cli/blob/b69af51c405418199c9ca46ad293208b984202b6/internal/pkg/manifest/validate.go#L1034

to be more of the format:
https://github.com/aws/copilot-cli/blob/b69af51c405418199c9ca46ad293208b984202b6/internal/pkg/manifest/validate.go#L792

Thank you 🥺

Reply:
Sure